cfgov/cfgov/settings/base.py

```diff
@@ -259,3 +260,14 @@
 
 # PDFReactor
 PDFREACTOR_LIB = os.environ.get('PDFREACTOR_LIB', '/opt/PDFreactor/wrappers/python/lib')
+
+# S3 Configuration
+if os.environ.get('S3_ENABLED', False):
+    DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
+    AWS_S3_SECURE_URLS = False  # use http instead of https
```

Comment:
This is a little confusing: `use http instead of https`. Does it mean a true value uses `http` or the reverse? Looks like the code is referring to the current state instead of the flag. Instead how about `True = use http; False = use https.`